python/sglang/bench_latency.py

```diff
@@ -200,16 +203,16 @@ def extend(reqs, model_runner):
         tree_cache=None,
     )
     batch.prepare_for_extend(model_runner.model_config.vocab_size)
-    output = model_runner.forward(batch, ForwardMode.EXTEND)
-    next_token_ids = batch.sample(output.next_token_logits)
-    return next_token_ids, output.next_token_logits, batch
+    logits_output = model_runner.forward(batch, ForwardMode.EXTEND)
+    sample_output = sampler.forward_cuda(logits_output, batch.sampling_info)
```

Comment:
Is this because Sampler supports torch.compile, and the issue arose from not updating it accordingly?